components/timezone/src/iana_ids.rs

```diff
@@ -105,10 +105,15 @@ impl<'a> IanaToBcp47MapperBorrowed<'a> {
     ///
     /// [ECMAScript Temporal]: https://tc39.es/proposal-temporal/#sec-isavailabletimezonename
     pub fn get(&self, iana_id: &str) -> Option<TimeZoneBcp47Id> {
+        self.get_bytes(iana_id.as_bytes())
+    }
+
+    #[doc(hidden)]
+    pub fn get_bytes(&self, iana_id: &[u8]) -> Option<TimeZoneBcp47Id> {
```

Comment:
Suggestion: this should be `get_utf8` and it can be public. Compare to `ComposingNormalizer::normalize` and `::normalize_utf8`

Reply:
This is more similar to our `from_bytes` methods than to `normalize_utf8`, as it's basically a raw tinystr

Reply:
This takes in a string and looks up in the data payload to find the corresponding tinystr if there is one. It is a data structure function. We're not doing any type conversions here so "from" is not accurate.

Besides, we're going to want `get_utf16` here as well at some point I think. That is, unless, according to #2413, we want to make these functions named `get`, `get8`, and `get16`? I think "UTF-16" is seen as more of an adjective though, and the reason `get_u32` was confusion to basically everyone was because "u32" is a noun.

Reply:
This is why I want to keep it doc-hidden for now, I don't want to block this PR on name bikeshedding.

Reply:
https://github.com/unicode-org/icu4x/issues/4368